pkg/scripts/download.go

```diff
@@ -44,14 +44,14 @@ func NewDownloader(proxyConfig *api.ProxyConfig) Downloader {
 // Returns information a boolean flag informing whether any download/copy operation
 // happened and an error if there was a problem during that operation
 func (d *downloader) Download(url *url.URL, targetFile string) (*api.SourceInfo, error) {
-	schemeReader := d.schemeReaders[url.Scheme]
+	r := d.schemeReaders[url.Scheme]
```

Comment:
The tools that shout the shadowing are not buggy in this case, it's just that it is a very subtle thing. Did you know about Method Values? We talked about this in one of our coding dojos some time ago, @php-coder must remember.

https://play.golang.org/p/C3DZmpy77S


Not saying we should blindly follow what linters suggest. In some cases shadowing is indeed the intended behavior. For example, when you have a variable in a closure with the same name as in the outer function, and they should be distinct variables -- I've seen it happen with the ubiquitous `err` variable name.

@bparees if you want this local variable stay as `schemeReader` so be it.